### PR TITLE
[INTEL MKL] DNNL1.x integration for tf_conv_ops.h and transpose.cc

### PR DESCRIPTION
**Pull request by Srini511**
DNNL1.x integration for tf_conv_ops.h and transpose.cc

### CODE REVIEW

File: tensorflow/core/kernels/mkl_transpose_op.cc [hunk 1]
```diff
@@ -16,16 +16,23 @@ limitations under the License.
...
+
 #define EIGEN_USE_THREADS
 
 #if !defined(INTEL_MKL_DNN_ONLY)
 #include "mkl_trans.h"
 #endif
 
+#ifdef ENABLE_MKLDNN_V1
+// MKL-DNN 1.0 renames TENSOR_MAX_DIMS (0.x) as MKLDNN_MAX_NDIMS
+#define TENSOR_MAX_DIMS MKLDNN_MAX_NDIMS
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl_transpose_op.cc]: Should this be in `mkl_types.h` instead?

File: tensorflow/core/kernels/mkl_transpose_op.cc [hunk 2]
```diff
@@ -141,12 +146,24 @@ Status MKLTransposeND(OpKernelContext* context, const Tensor& in_tensor,
...
+    net.push_back(reorder(in.GetOpMem(), out.GetOpMem()));
+    std::vector<std::unordered_map<int, memory>> net_args;
+    net_args.push_back(
+        {{MKLDNN_ARG_FROM, in.GetOpMem()}, {MKLDNN_ARG_TO, out.GetOpMem()}});
+    net.at(net_idx).execute(*transpose_stream, net_args.at(net_idx));
+#else
     net.push_back(FindOrCreateReorder<T>(in.GetUsrMem(), out.GetUsrMem()));
-    stream(stream::kind::eager).submit(net).wait();
+    transpose_stream->submit(net).wait();
+#endif
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl_transpose_op.cc]: Nit:
```suggestion
#endif  // ENABLE_MKLDNN_V1
```

**Comment by Srini511** [comment]
@penpornk Gentle reminder for review.

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for the PR! I have minor comments.

File: tensorflow/core/kernels/mkl_transpose_op.cc [hunk 3]
```diff
@@ -158,7 +153,7 @@ Status MKLTransposeND(OpKernelContext* context, const Tensor& in_tensor,
 #else
     net.push_back(FindOrCreateReorder<T>(in.GetUsrMem(), out.GetUsrMem()));
     transpose_stream->submit(net).wait();
-#endif
+#endif //ENABLE_MKLDNN_V1
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl_transpose_op.cc]: Nit: two spaces after `#endif` and one space after `//`.
```suggestion
#endif  // ENABLE_MKLDNN_V1
```

**Comment by Srini511** [comment]
@penpornk done the changes.

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for the quick fixes! The formatting fix is still not quite right. Sorry for the nit!

File: tensorflow/core/kernels/mkl_transpose_op.cc (hunk 3 above)
Comment (Srini511) [reply tensorflow/core/kernels/mkl_transpose_op.cc]: How do i just accept the change?

Comment (penpornk) [reply tensorflow/core/kernels/mkl_transpose_op.cc]: You can just click the `commit suggestion` button, I think. I haven't seen the change reflected on my side, btw.

**Comment by Srini511** [comment]
@penpornk Done.

File: tensorflow/core/kernels/mkl_transpose_op.cc (hunk 3 above)
Comment (penpornk) [reply tensorflow/core/kernels/mkl_transpose_op.cc]: Oh I see it now.

**Review by penpornk — APPROVED** [review]
Thank you!

**Pull request merged**